pom.xml

```diff
@@ -121,9 +122,9 @@
   <dependencies>
         <!-- Spigot -->
         <dependency>
-          <groupId>com.destroystokyo.paper</groupId>
+          <groupId>io.papermc.paper</groupId>
           <artifactId>paper-api</artifactId>
-          <version>${spigot.version}</version>
+            <version>${spigot.version}</version>
```

Comment:
Well.. you kinda fixed it :stuck_out_tongue_closed_eyes: can you also fix the extra whitespace you added? :wink: